docs/integration/alerts-servicenow-integration.md

```diff
@@ -0,0 +1,57 @@
+title: ServiceNow Alerts Integration
+description: Sematext threshold, anomaly and/or heartbeat Alerts integration with ServiceNow. Use them together to get the information about each notification quickly and on any device.
+
+## In ServiceNow
+
+**1.** In the ServiceNow Service Management screen, search and select the option "Scripted REST APIs"
+
+<img class="content-modal-image" alt="Create Service Now Integration - Search for Scripted REST APIs" src="../../images/integrations/create-servicenow-integration-search.png" title="Create Service Now Integration - Search for Scripted REST APIs">
+
+**2.** Create a new Scripted REST API by clicking **New** on the top bar above the table.
+
+<img class="content-modal-image" alt="Create Service Now Integration - Scripted REST APIs" src="../../images/integrations/create-servicenow-integration-scripted-apis.png" title="Create Service Now Integration - Scripted REST APIs">
+
+**3.** Fill out the name and API ID and then click **Submit**.
+
+<img class="content-modal-image" alt="Create Service Now Integration - Scripted REST API Creation" src="../../images/integrations/create-servicenow-integration-scripted-api-creation.png" title="Create Service Now Integration - Scripted REST API Creation">
+
+**4.** Once you've created the API, you will be taken to the initial Scripted Rest API screen. You can find your application by entering the app name in the search bar and clicking Enter on your keyboard. The app should be shown in the results. Click on the name of the application to continue the setup.
```

Comment:
is "app name" referring to "App" in Sematext?

Reply:
Yes

Reply:
Nope, it's the app from Service now or API to be more exact, I'll make the propper adjustments